shared/versions/stack/README

```diff
@@ -0,0 +1,29 @@
+Books that are "part of the stack" include these files by looking at their
+branch. So the docs for Elasticsearch can use this:
+
+```
+include::{asciidoc-dir}/../../shared/versions/stack/{source_branch}.asciidoc[]
+```
+
+Which will resolve to the `6.3.asciidoc` file for he 6.3 branch. Or the
+`7.2.asciidoc` file for the 7.2 branch. Or `master.asciidoc` for the master
+branch.
+
+NOTE: `current.asciidoc` is never automatically used.
+
+Books outside of the stack have three choices:
+
+1. Don't use these versions files at all.
```

Comment:
Hmmm, I don't think this is really an option any more. With the shared version change, if I want to use any of the shared attributes in the APM Agents, they also **have** to use the shared version files. An example of why is in the recent ECS PR: https://github.com/elastic/docs/pull/1170.

Reply:
So, ok, _technically_ it is an option still. But I would have to start setting attribute values in each of the Agent's `index.asciidoc` files.

Reply:
Maybe I should just drop it from the list then.

Reply:
I think so.